### DISCUSSION TITLE
Disable notifications?

### DISCUSSION THREAD

**Discussion by Maltimore** [tuskyapp/Tusky#2452] [General]
Hi,
I think it should be really easy to disable notifications, however I can't find the setting! I never want Tusky to make a popup in the Android dropdown screen and I also never want sounds. How do I achieve that?
Edit: I do still want to see the notifications in the notifications tab when I'm in the app.

**Comment by charlag** [comment]
Hey, account preferences -> notifications will get you to system settings for notifications

**Reply by Maltimore** [reply]
Ah, interesting! Thank you!
I guess I never suspected it would be under `account preferences`, and only looked under `preferences`. I wonder: what is the deciding factor for which settings go to `account preferences`, and which go to `preferences`?

**Reply by charlag** [reply]
The factor is simple: what is different for different accounts on the device is under account preferences.
